tensorflow/python/BUILD

```diff
@@ -3001,7 +3001,7 @@ cuda_py_test(
 
 cuda_py_tests(
     name = "training_tests",
-    size = "small",
+    size = "medium",
```

Comment:
Is it SYCL being slow that means you have to up the timeout? or is it just because you are now running the tests multiple times?